BESMod/unitTests-dymola.log

```diff
@@ -0,0 +1,2 @@
+Regression tests are only run for the following package:
```

Comment:
When was this added?

Reply:
This is added by the CI. Should I remove it and add it to gitignore?